specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/networkManagerSecurityConfiguration.json

```diff
@@ -0,0 +1,1349 @@
+    },
+    "SecurityConfigurationAdminRuleListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecurityConfigurationAdminRule"
+          },
+          "description": "A list of managed network security configuration admin rules"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets the URL to get the next set of results."
+        }
+      },
+      "description": "Security Configuration Admin Rule List Result."
+    },
+    "SecurityConfigurationUserRuleListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecurityConfigurationUserRule"
+          },
+          "description": "A list of network manager security configuration user rules"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets the URL to get the next set of results."
+        }
+      },
+      "description": "Security Configuration User Rule List Result."
+    },
+    "SecurityConfigurationRuleListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecurityConfigurationRule"
+          },
+          "description": "A list of network manager security configuration rules"
+        },
+        "failedImports": {
+          "$ref": "#/definitions/FailedImports",
+          "description": "Failed Imports."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets the URL to get the next set of results."
+        }
+      },
+      "description": "Security Configuration Rule List Result."
+    },
+    "SecurityConfigurationAdminRule": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SecurityConfigurationRuleAdminPropertiesFormat",
+          "x-ms-client-flatten": true,
+          "description": "Indicates the properties of the security admin rule"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Network security admin rule."
+    },
+    "SecurityConfigurationRuleAdminPropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "A friendly name for the rule."
+        },
+        "description": {
+          "type": "string",
+          "description": "A description for this rule. Restricted to 140 chars."
+        },
+        "protocol": {
+          "type": "string",
+          "description": "Network protocol this rule applies to.",
+          "enum": [
+            "Tcp",
```

Comment:
Shouldn't these be in all caps as these are acronyms?

Reply:
We are keeping it as same as protocol in networksecuritygroup.json